src/PostmarkTransport.php

```diff
@@ -54,16 +55,21 @@ public function getApiEndpoint(Swift_Mime_SimpleMessage $message): string
      * @return int
      *
      * @throws \GuzzleHttp\Exception\GuzzleException
+     * @throws Swift_TransportException
      */
     public function send(Swift_Mime_SimpleMessage $message, &$failedRecipients = null): int
     {
         $this->beforeSendPerformed($message);
 
-        $response = $this->client->request(
-            'POST',
-            $this->getApiEndpoint($message),
-            $this->payload($message)
-        );
+        try {
+            $response = $this->client->request(
+                'POST',
+                $this->getApiEndpoint($message),
+                $this->payload($message)
+            );
+        } catch (\GuzzleHttp\Exception\ConnectException | \GuzzleHttp\Exception\ServerException $exception) {
```

Comment:
In order to not force the PHP 8 minimum version, could we instead have multiple catch blocks? The plan is to eventually drop older versions of PHP, but I don't think something like this warrants the break. In addition, could you import the class names?

Reply:
Hey Paul, thanks for reviewing this PR so fast! :)  

The pipe character for exceptions is something that was introduced in PHP 7.1, so this should be ok :) 
I tested this in a PHP 7.4 environment by the way. If you still prefer having multiple catch blocks, then I can change that, no problem. 

I usually import all class names in my own projects, but as I saw that it wasn't done for the @throws annotation (\GuzzleHttp\Exception\GuzzleException) in the docblock , I thought it wasn't necessary. Anyway, I've added the imports now :)

Reply:
Ah! So it was. I assumed that was part of the union type stuff in PHP 8 👍